### PR TITLE
Even more commands

### PR DESCRIPTION
**Pull request by khaosdoctor**
Added more commands, but there's a need to reorganize them

### CODE REVIEW

**Comment by darlanmendonca** [comment]
Nice commands :D

but man, lets organize the order of descriptions in same order of main index please, Is hard to view diffs unordened

**Comment by khaosdoctor** [comment]
I didn't understand, you mean the order of the topics?

**Comment by darlanmendonca** [comment]
yep, the topics in your PR is different in order of content

File: README.md [hunk 1]
```diff
@@ -15,6 +15,19 @@ Below you find some help to commands that you will probaly need.
...
+- [TAIL](#tail)
+- [FIND](#find)
+- [RMDIR](#rmdir)
+- [UNLINK](#unlink)
+- [MV](#mv)
+- [CP](#cp)
+- [SUDO](#sudo)
+- [TRACEROUTE](#traceroute)
+- [KILL](#kill)
+- [LN](#ln)
```
Comment (darlanmendonca) [inline README.md:30]: here, LN is the last

File: README.md [hunk 2]
```diff
@@ -207,3 +220,149 @@ Shows the full description of a command.
...
+```
+
+Can also be used to create a copy of files in the `pwd` if you specify a __file__ name and not a __folder__ name.
+
+```sh
+# Creates a second file called "b.md" from "a.md"
+cp a.md b.md
+```
+
+### MV
```
Comment (darlanmendonca) [inline README.md:353]: but here, in content, the last is MV, instead LN

**Comment by darlanmendonca** [comment]
the target is made it more easy the code review :D

**Comment by khaosdoctor** [comment]
Yep, I got it, will work on this ASAP

**Comment by khaosdoctor** [comment]
Will close this because I forgot to add some stuff in my repo here

**Pull request closed without merging**